### PR TITLE
Add Wally.app Latest

### PR DESCRIPTION
**Pull request by fdidron**
Wally is an open source application allowing users to flash new
firmwares to their ZSA keyboards.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/wally.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'wally' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/wally.rb]: Set this (`1.1.1`) and `sha256`.

File: Casks/wally.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
+
+  url 'https://configure.ergodox-ez.com/wally/osx'
```
Comment (vitorgalvao) [inline Casks/wally.rb]: Use the download from https://github.com/zsa/wally/releases, don’t forget [the comment](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) (`github.com/zsa/wally …`).

Comment (vitorgalvao) [reply Casks/wally.rb]: Also add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Welcome. This just needs a few fixes, and we can merge it.

**Comment by fdidron** [comment]
Thanks @vitorgalvao for the pointers, I fixed the issues you reported.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Submit to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers) instead, please.

**Pull request closed without merging**